### PR TITLE
www-servers/caddy: version bump to 2.2.1

### PR DESCRIPTION
**Pull request by epsilon-0**
Package-Manager: Portage-3.0.8, Repoman-3.0.1
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

---
not taking the package

### CODE REVIEW

**Comment by rossbridger** [comment]
That's quite a lot of mods to download, is it possible to package them into a single tarball?

File: www-servers/caddy/caddy-2.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,991 @@
...
+	"sigs.k8s.io/yaml v1.1.0/go.mod"
+	"sourcegraph.com/sourcegraph/go-diff v0.5.0/go.mod"
+	"sourcegraph.com/sqs/pbtypes v0.0.0-20180604144634-d3ebe8f20ae4/go.mod"
+	"sourcegraph.com/sqs/pbtypes v1.0.0/go.mod"
+	)
+go-module_set_globals
+SRC_URI="https://github.com/caddyserver/caddy/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	${EGO_SUM_SRC_URI}"
+
+LICENSE="Apache-2.0"
```
Comment (juippis) [inline www-servers/caddy/caddy-2.2.0.ebuild:978]: This looks good, but you need to include licenses for all included libs in this ebuild. 
https://packages.gentoo.org/packages/search?q=go-license-detector there are bunch of tools to help you.

**Review by juippis — COMMENTED** [review]
@rossbridger yeah, **sigh** that's just how go works in gentoo :)

File: www-servers/caddy/caddy-2.2.0.ebuild (hunk 1 above)
Comment (epsilon-0) [reply www-servers/caddy/caddy-2.2.0.ebuild:978]: Oh damn. I thought go ego vendor does that like cargo ebuild.
will do this soon. Thanks.

Comment (epsilon-0) [reply www-servers/caddy/caddy-2.2.0.ebuild]: done

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**